apps/meteor/client/views/room/contextualBar/RoomMembers/InviteUsers/InviteUsersLoading.tsx

```diff
@@ -0,0 +1,17 @@
+import { Skeleton } from '@rocket.chat/fuselage';
+import type { ReactElement } from 'react';
+
+import InviteUsersWrapper from './InviteUsersWrapper';
+
+type InviteUsersProps = {
+	onClose: () => void;
+	onClickBack: (() => void) | undefined;
+};
+
+const InviteUsersLoading = ({ onClose, onClickBack: onClickBackMembers }: InviteUsersProps): ReactElement => (
+	<InviteUsersWrapper onClose={onClose} onClickBack={onClickBackMembers}>
+		<Skeleton w='full' />
```

Comment:
what if we try to render something similar to the `ContextualbarSkeleton`?

Reply:
`ContextualbarSkeleton` is loading the whole contextualbar - header title and content - and has no `Back` or `Close` buttons.
In this case we have all these info loaded already, so I decided to put a skeleton only on the content that is pending.